applications/connecting_applications_to_services/odc-connecting-service-to-application-using-the-developer-perspective.adoc

```diff
@@ -0,0 +1,84 @@
+[id="odc-connecting-service-to-application-using-the-developer-perspective"]
+= Connecting a service to an application using the Developer perspective
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: odc-connecting-service-to-application-using-the-developer-perspective
+
+In addition to grouping multiple components within an application, you can also use the *Topology* view to connect components with each other. You can either use a binding connector or a visual one to connect components.
+
+A binding connection between the components can be established only if the target node is an Operator-backed service. This is indicated by the *Create a binding connector* tool-tip which appears when you drag an arrow to such a target node. When an application is connected to a service using a binding connector a `ServiceBinding` resource is created. Then, the Service Binding Operator controller projects the necessary binding data into the application deployment. After the request is successful, the application is redeployed establishing an interaction between the connected components.
+
+A visual connector establishes only a visual connection between the components, depicting an intent to connect. No interaction between the components is established. If the target node is not an Operator-backed service the *Create a visual connector* tool-tip is displayed when you drag an arrow to a target node.
+
+== Creating a visual connection between components
+
+You can depict an intent to connect application components using the visual connector.
+
+This procedure walks through an example of creating a visual connection between a PostgreSQL Database service and a Spring PetClinic sample application.
+
+.Prerequisites
+
+* Ensure that you have created and deployed a Spring PetClinic sample application using the *Developer* perspective.
+* Ensure that you have created and deployed a Crunchy PostgreSQL database instance using the *Developer* perspective.
```

Comment:
is there details in the doc how to do this step?

Reply:
Will provide links to Getting started with service binding topics under additional resources section.